src/command_modules/azure-cli-hdinsight/azure/cli/command_modules/hdinsight/custom.py

```diff
@@ -6,14 +6,15 @@
 from knack.log import get_logger
 from knack.util import CLIError
 from azure.cli.core.util import sdk_no_wait
+from ._client_factory import cf_hdinsight_script_actions, cf_hdinsight_script_execution_history
 
 
 logger = get_logger(__name__)
 
 
 # pylint: disable=too-many-locals, too-many-branches, too-many-statements
-def create_cluster(cmd, client, cluster_name, resource_group_name, location=None, tags=None, no_wait=False,
-                   cluster_version='default', cluster_type='spark', cluster_tier=None,
+def create_cluster(cmd, client, cluster_name, resource_group_name, cluster_type, location=None, tags=None,
```

Comment:
Why are we removing the default type and making it required?

Reply:
This is by design. We think that could be dangerous going forward if we decide we want to change the default. So we prefer to removing default values.

Reply:
Let's try not to do this too often in the future, as this will break user scripts.
Can you also add this to the history and indicate that it is breaking?

Reply:
Good suggestion! Added this to the history.